main.mk

```diff
@@ -55,7 +55,11 @@ endif
 %.xml: %.org
 	$(oxtradoc) -m outline-to-xml -n "$@" $< > $@
 
-XML2RFCV3OPTION := $(if $(V3),--v3)
+ifeq (true,$(XML2RFCV3))
+XML2RFCV3 := --v3
```

Comment:
This is a little odd, overwriting the value of the tested variable, but it does work.

Reply:
Oh, I thought that's what you'd suggested in https://github.com/martinthomson/i-d-template/pull/195#discussion_r332904139. I don't have any particular opinion about the best way to do this. :)

Reply:
I must have typed the same thing twice :/  I'll sort it out.